examples/hook.php

```diff
@@ -22,46 +30,48 @@
     // Create Telegram API object
     $telegram = new Longman\TelegramBot\Telegram($API_KEY, $BOT_NAME);
 
-    //// Enable MySQL
-    //$telegram->enableMySQL($mysql_credentials);
+    // Error, Debug and Raw Update logging
+    //Longman\TelegramBot\TelegramLog::initialize($your_external_monolog_instance);
+    //Longman\TelegramBot\TelegramLog::initErrorLog($path . '/' . $BOT_NAME . '_error.log');
+    //Longman\TelegramBot\TelegramLog::initDebugLog($path . '/' . $BOT_NAME . '_debug.log');
+    //Longman\TelegramBot\TelegramLog::initUpdateLog($path . '/' . $BOT_NAME . '_update.log');
+
+    // Enable MySQL
+    //$telegram->enableMySql($mysql_credentials);
 
-    //// Enable MySQL with table prefix
-    //$telegram->enableMySQL($mysql_credentials, $BOT_NAME . '_');
+    // Enable MySQL with table prefix
+    //$telegram->enableMySql($mysql_credentials, $BOT_NAME . '_');
 
-    //// Add an additional commands path
+    // Add an additional commands path
     //$telegram->addCommandsPath($commands_path);
 
-    //// Enable admin users
+    // Enable admin user(s)
     //$telegram->enableAdmin(your_telegram_id);
+    //$telegram->enableAdmins([your_telegram_id, other_telegram_id]);
 
-    //// Add the channel you want to manage
+    // Add the channel you want to manage
     //$telegram->setCommandConfig('sendtochannel', ['your_channel' => '@type_here_your_channel']);
 
-    //// Here you can set some command specific parameters,
-    //// for example, google geocode/timezone api key for date command:
+    // Here you can set some command specific parameters,
+    // for example, google geocode/timezone api key for /date command:
     //$telegram->setCommandConfig('date', ['google_api_key' => 'your_google_api_key_here']);
 
-    //// Logging
-    //\Longman\TelegramBot\TelegramLog::initialize($your_external_monolog_instance);
-    //\Longman\TelegramBot\TelegramLog::initErrorLog($path . '/' . $BOT_NAME . '_error.log');
-    //\Longman\TelegramBot\TelegramLog::initDebugLog($path . '/' . $BOT_NAME . '_debug.log');
-    //\Longman\TelegramBot\TelegramLog::initUpdateLog($path . '/' . $BOT_NAME . '_update.log');
-
-    //// Set custom Upload and Download path
+    // Set custom Upload and Download path
     //$telegram->setDownloadPath('../Download');
     //$telegram->setUploadPath('../Upload');
 
-    //// Botan.io integration
+    // Botan.io integration
     //$telegram->enableBotan('your_token');
 
     // Handle telegram webhook request
     $telegram->handle();
 } catch (Longman\TelegramBot\Exception\TelegramException $e) {
-    // Silence is gold!
-    // echo $e;
-    // log telegram errors
-    \Longman\TelegramBot\TelegramLog::error($e);
+    // Silence is golden!
```

Comment:
ahahah
http://forum.wordreference.com/threads/silence-is-gold-or-golden.719876/